packages/babel-types/src/definitions/core.ts

```diff
@@ -663,20 +674,22 @@ defineType("RegExpLiteral", {
       validate: assertValueType("string"),
     },
     flags: {
-      validate: chain(
-        assertValueType("string"),
-        Object.assign(
-          function (node, key, val) {
-            if (!process.env.BABEL_TYPES_8_BREAKING) return;
-
-            const invalid = /[^gimsuy]/.exec(val);
-            if (invalid) {
-              throw new TypeError(`"${invalid[0]}" is not a valid RegExp flag`);
-            }
-          } as Validator,
-          { type: "string" },
-        ),
-      ),
+      validate: process.env.BABEL_8_BREAKING
+        ? chain(
+            assertValueType("string"),
+            Object.assign(
+              function (node, key, val) {
+                const invalid = /[^gimsuy]/.exec(val);
```

Comment:
Something seems wrong here, at least one has a `v`?

Reply:
`v` and `d`

Reply:
https://tc39.es/ecma262/#sec-isvalidregularexpressionliteral